### PR TITLE
fix: colons in scripts

### PR DESCRIPTION
**Pull request by mistermalm**
## **Describe pull-request**  
I like colons

## **How to test**  
1. Stay in tegel or cd into core
2. Run scripts that changed

### CODE REVIEW

**Review by timrombergjakobsson — APPROVED** [review]

File: packages/core/package.json [hunk 1]
```diff
@@ -373,16 +373,16 @@
     "start": "stencil build --dev --watch --serve",
     "start:dev": "stencil build --dev --watch --serve",
     "test": "node docker-test-runner.js",
-    "update-snapshots": "docker run --rm --network host -v $(pwd):/work/ -w /work/ mcr.microsoft.com/playwright:v1.46.1-jammy /bin/bash -c 'npm install && npx playwright test --update-snapshots && exit'",
+    "update:snapshots": "docker run --rm --network host -v $(pwd):/work/ -w /work/ mcr.microsoft.com/playwright:v1.46.1-jammy /bin/bash -c 'npm install && npx playwright test --update-snapshots && exit'",
     "test.watch": "stencil test --spec --e2e --watchAll",
     "generate": "stencil generate",
-    "copy-assets": "node scripts/copy-assets.mjs",
+    "copy:assets": "node scripts/copy-assets.mjs",
```
Comment (theJohnnyMe) [inline packages/core/package.json:379]: Please update the documentation of coping local assets that mention this command or revert this change.

**Review by theJohnnyMe — CHANGES_REQUESTED** [review]

File: packages/core/package.json (hunk 1 above)
Comment (timrombergjakobsson) [reply packages/core/package.json:379]: @mistermalm have you seen this?

Comment (mistermalm) [reply packages/core/package.json:379]: So if you mean "static-assets-copying.md" by documentation, this readme file includes the docs for npx script and not npm. 

In other words, "npx copy-assets" is still correct in the docs, but npm uses colon

So question, should we revert this change? Don't really like to mix colons and dashes

Comment (Lunkan89) [reply packages/core/package.json:379]: I believe he is just referring to this line:
https://arc.net/l/quote/tcleoaul

Comment (mistermalm) [reply packages/core/package.json:379]: Okay, but then I don't need to change anything

**Review by timrombergjakobsson — APPROVED** [review]

**Review by nathalielindqvist — APPROVED** [review]
:shipit:
